src/lib/builder/net-builder/net-builder.ts

```diff
@@ -51,6 +52,7 @@ export class NetBuilderClass implements NetBuilder {
         is_digital_signal: this.props.is_digital_signal,
         is_power: this.props.is_power,
         is_ground: this.props.is_ground,
+        trace_width: this.props.trace_width,
```

Comment:
to fix this error, i think you'll need to add `trace_width` to `source_net` on [tscircuit/soup](https://github.com/tscircuit/soup)

Reply:
ahh you did https://github.com/tscircuit/soup/pull/17#pullrequestreview-2210780920

Reply:
Yeah, have made another PR for it